### PR TITLE
Almost finished Irish translation

### PR DESCRIPTION
**Pull request by seananocoistin**
I have translated 92% of the Irish translation. The remainder shall be done soon.

### CODE REVIEW

**Comment by a-polivanchuk** [comment]
Hi,
If you see the Travis CI check output, there are some error messages:
#   Failed test 'locales/ga_IE/LC_MESSAGES/duckduckgo.po - specifier "%sReset filters %s" matches "%Athshocraigh na scagairí %s"'
==You need to review the string "%sReset filters %s" for possible issues.==

#   Failed test 'Translation for "More Info" in locales/ga_IE/LC_MESSAGES/duckduckgo.po should not be changed : "Tuilleadh eolais"'
#   Failed test 'Translation for "Viewing ads is privacy protected by DuckDuckGo. Ad clicks are managed by Microsoft's ad network" in locales/ga_IE/LC_MESSAGES/duckduckgo.po should not be changed : "Má bhreathnaítear ar fhógraí, cosnaíonn DuckDuckGo do phríobháideachas. Is líonra fógraíochta Microsoft a bhainistíonn cliceanna ar fhógraí"'
==These two strings should be kept without any changes. Just copy the whole strings from the current master and it should resolve the issue.==

File: locales/ga_IE/LC_MESSAGES/duckduckgo.po [hunk 1]
```diff
@@ -106,11 +100,11 @@ msgstr ""
 
 msgctxt "precise_user_location"
 msgid "%sReload%s DuckDuckGo, and try again."
-msgstr ""
+msgstr "%sAthluchtaigh%s DuckDuckGo, agus féach arís é."
 
 msgctxt "noresults"
 msgid "%sReset filters %s"
-msgstr ""
+msgstr "%Athshocraigh na scagairí %s"
```
Comment (GioSensation) [inline locales/ga_IE/LC_MESSAGES/duckduckgo.po:107]: Fix typo.

```suggestion
msgstr "%sAthshocraigh na scagairí %s"
```

File: locales/ga_IE/LC_MESSAGES/duckduckgo.po [hunk 2]
```diff
@@ -5063,18 +4906,16 @@ msgid "View on Map"
 msgstr "Breathnaigh air ar an Léarscáil"
 
 msgctxt "Ads GDPR, internal use only. Do not change"
-msgid ""
-"Viewing ads is privacy protected by DuckDuckGo. Ad clicks are managed by "
-"Microsoft's ad network"
+msgid "Viewing ads is privacy protected by DuckDuckGo. Ad clicks are managed by Microsoft's ad network"
 msgstr ""
+"Má bhreathnaítear ar fhógraí, cosnaíonn DuckDuckGo do phríobháideachas. Is líonra fógraíochta Microsoft a "
+"bhainistíonn cliceanna ar fhógraí"
```
Comment (GioSensation) [inline locales/ga_IE/LC_MESSAGES/duckduckgo.po:4912]: String for internal use only. Reverting.

```suggestion
```

File: locales/ga_IE/LC_MESSAGES/duckduckgo.po [hunk 3]
```diff
@@ -2789,7 +2700,7 @@ msgstr "Tuilleadh eolais"
 
 msgctxt "Ads GDPR, internal use only. Do not change"
 msgid "More Info"
-msgstr ""
+msgstr "Tuilleadh eolais"
```
Comment (GioSensation) [inline locales/ga_IE/LC_MESSAGES/duckduckgo.po:2703]: String for internal use only. Reverting.

```suggestion
msgstr ""
```

**Comment by GioSensation** [comment]
Quite a few conflicts here. I tried resolving them to the best of my abilities. Again, apologies for the delay, but this is now merged and will be released in the near future. Thanks @seananocoistin!

**Pull request merged**